shared/js/background/chrome-events.es6.js

```diff
@@ -70,10 +88,25 @@ chrome.webRequest.onHeadersReceived.addListener(
             // returns a promise
             return ATB.updateSetAtb(request)
         }
+
+        if (blockingExperimentActive()) {
+            let responseHeaders = request.responseHeaders
+            // Strip 3rd party response header
+            const tab = tabManager.get({ tabId: request.tabId })
+            if (!request.responseHeaders) return
+            if (tab && tab.site.whitelisted) return
+            if (tab && utils.isFirstParty(request.url, tab.url)) return
+            if (!cookieConfig.isExcluded(request.url) && trackerutils.isTracker(request.url)) {
+                responseHeaders = responseHeaders.filter(header => header.name.toLowerCase() !== 'set-cookie')
+            }
+
+            return { responseHeaders: responseHeaders }
```

Comment:
```suggestion
            if (!cookieConfig.isExcluded(request.url) && trackerutils.isTracker(request.url)) {
                responseHeaders = responseHeaders.filter(header => header.name.toLowerCase() !== 'set-cookie')
                return { responseHeaders }
            }
            ```

This avoids returning if response headers are unchanged.